security.md

```diff
@@ -66,67 +76,91 @@ Deprecated, do not use.
 
 <h4 id="expiration"><a href="#expiration">expiration</a></h4>
 
-Deprecated, do not use.
+See [vc-data-model](https://www.w3.org/TR/vc-data-model/#expiration).
```

Comment:
The example in this link describes the `expirationDate` property, not `expiration`

Reply:
I think they are synonyms... its hard to tell..